core/callable_method_pointer.h

```diff
@@ -151,11 +151,11 @@ void call_with_variant_args(T *p_instance, void (T::*p_method)(P...), const Vari
 		return;
 	}
 
-	if ((size_t)p_argcount < sizeof...(P)) {
+	/*if ((size_t)p_argcount < sizeof...(P)) { // TODO: Some trick to replace sizeof...(P) to get "minimum arguments"?
```

Comment:
I think we need to put "minimum arguments" here, and not exactly what we need.

This create inconsistency because when we have a default value on a function, and create make_binds for fill those arguments. And if we change one of them, the other will stay equal generating a inconsistency.

Just I trying to figure out how to check properly. But this is workig.